packages/frontend/src/components/common/Table/BodyCell.tsx

```diff
@@ -91,21 +110,17 @@ const BodyCellWrapper: FC<CommonBodyCellProps> = ({ onSelect, ...props }) => {
             renderTarget={({ ref }) => (
                 <BodyCell
                     {...props}
-                    hasContextMenu
-                    isSelected={true}
-                    onSelect={handleCellSelect}
+                    style={
+                        isCellSelected
+                            ? { position: 'relative', zIndex: 21 }
```

Comment:
Issue where the cell is on top of the underlying data modal.

![Screenshot 2022-11-14 at 14 34 31](https://user-images.githubusercontent.com/9117144/201687369-9f7d0415-451d-43cd-a3d7-de47be640370.png)

Reply:
good catch 👍